db/agents.json

```diff
@@ -356,16 +357,6 @@
         "Clementine 1.3.1"
       ]
     },
-    {
-      "regex": "^Dalvik.+Android",
-      "name": null,
-      "type": "Mobile App",
-      "os": "Android",
-      "examples": [
-        "Dalvik/2.1.0 (Linux; U; Android 8.0.0; SM-G950U Build/R16NW)",
-        "Dalvik/1.6.0 (Linux; U; Android 4.4.2; SM-N900 Build/KOT49H)"
-      ]
-    },
```

Comment:
NPR One android now starts with `Dalvik` ... to prevent this one from matching, I had to move it down towards the bottom of this list.